src/lib/command-helpers.mjs

```diff
@@ -114,28 +122,44 @@ const COMMON_CONFIGURABLE_QUERY_OPTIONS = {
     default: 5000,
     group: "API:",
   },
-  summary: {
-    type: "boolean",
-    description:
-      "Output the summary field of the API response or nothing when it's empty. Only applies to v10 queries.",
-    default: false,
-    group: "API:",
-  },
   performanceHints: {
     type: "boolean",
     description:
-      "Output the performance hints for the current query or nothing when no hints are available. Only applies to v10 queries.",
+      "Output the performance hints for the current query or nothing when no hints are available. Only applies to v10 queries. Sets '--include summary'",
     default: false,
     group: "API:",
   },
+  include: {
+    type: "array",
+    choices: ["all", ...QUERY_INFO_CHOICES],
+    default: [],
```

Comment:
this might be spicy, but is this major version upgrade the right time to change the default behavior?

if we're targeting primarily human users, should it be `[summary, stats]`?

Reply:
hmm I think I could get behind summary as a default. most queries don't have a summary, but having it on would make sure things like `dbg`, `log`, and any warning spit out by core appear by default.

I definitely want to se stats some times. I definitely don't want to see every time. Maybe

Shall we run with default as `["summary"]`?

Reply:
I will be switching over to doing more in CLI for random sandbox/test queries where I would most often go to the dashboard. That's the one case where I might want stats more often than not. I'll probably configure that profile in my global config to show stats every time.

Reply:
with default and "none" option
<img width="1007" alt="image" src="https://github.com/user-attachments/assets/26a443c9-cad6-43a7-afe6-be9edd3c371b" />